assets/src/components/searchForm.tsx

```diff
@@ -257,64 +343,75 @@ export const SearchForm = ({
           >
             <CircleXIcon />
           </button>
-          <button
-            type="submit"
-            title="Submit"
-            className="c-search-form__submit"
-            onClick={onSubmit}
-            // TODO(design): add error states instead of using `disabled`
-            disabled={!isValidSearchText(inputText)}
-          >
-            <SearchIcon />
-          </button>
+          {comboboxType == "map_search" && (
+            <button
+              type="submit"
+              title="Submit"
+              className="c-search-form__submit"
+              onClick={onSubmit}
+              // TODO(design): add error states instead of using `disabled`
+              disabled={!isValidSearchText(inputText)}
+            >
+              <SearchIcon />
+            </button>
+          )}
+          {comboboxType == "select" && (
+            <button
+              // TODO: rename this class (follow-up PR)
+              // TODO: for some reason, this button is still submitting???
```

Comment:
> // TODO: for some reason, this button is still submitting???

you likely need `type="button"`

Reply:
I didn't expect this! Nailed it, though.